uiowa/uiowa
# I1922 hide si widget

This resolves #1922 and should collapse the Site Improve widget when editing the layout of a layout builder page.

To test:
1. Pull down branch and set up a Layout builder site.
2. Go to any layout builder page and edit its layout.
3. Check to make sure the SI widget is collapsed, and that a double click opens it.
